include/merlin/types.cuh

```diff
@@ -16,8 +16,8 @@
 
 #pragma once
 
-#include <stddef.h>
```

Comment:
Pls keep it, this header is for `size_t`.

Reply:
Numeric limits imports it in turn. But well, if it makes you happy - i can put it back.